packages/shared/lib/services/file/local.service.ts

```diff
@@ -48,12 +48,10 @@ class LocalFileService {
     public putIntegrationFile(syncName: string, fileContents: string, distPrefix: boolean) {
         try {
             const realPath = fs.realpathSync(process.env['NANGO_INTEGRATIONS_FULL_PATH'] as string);
-            if (!fs.existsSync(`${realPath}${distPrefix ? '/dist' : ''}/${syncName}`)) {
```

Comment:
why is this being deleted?

Reply:
Because in an upgrade scenario we want to overwrite the file so the check for the local file existence is no needed. Also in an install scenario it would be OK to overwrite the file as well.